iOS/ViewControllers/Courses/PeerAssessmentHintViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import UIKit
+
+class PeerAssessmentHintViewController: UIViewController {
+
+    @IBOutlet private weak var peerAssessmentInfoView: UIStackView!
+    @IBOutlet private weak var deadlineMessageView: UIStackView!
+    @IBOutlet private weak var assessmentTitleLabel: UILabel!
+    @IBOutlet private weak var assessmentTypeLabel: UILabel!
+    @IBOutlet private weak var assessmentInstructionsView: UITextView!
+    @IBOutlet private weak var assessmentPointsLabel: UILabel!
+    @IBOutlet private weak var peerAssessmentTypeLabel: UILabel!
+    @IBOutlet private weak var peerAssessmentTypeImage: UIImageView!
+    @IBOutlet private weak var noteLabel: UILabel!
+    @IBOutlet private weak var redirectButton: UIButton!
+    @IBOutlet private weak var deadlineLabel: UILabel!
+    @IBOutlet private weak var deadlineDateView: UIStackView!
+
+    private static let pointsFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.decimalSeparator = "."
+        formatter.minimumFractionDigits = 1
+        formatter.maximumFractionDigits = 1
+        return formatter
+    }()
+
+    private static let dateFormatter = DateFormatter.localizedFormatter(dateStyle: .long, timeStyle: .short)
+
+    public var formattedDateWithTimeZone: String? {
+        guard let date = self.courseItem.deadline else {
+            return nil
+        }
+
+        var dateText = Self.dateFormatter.string(from: date)
+        if let timeZoneAbbreviation = TimeZone.current.abbreviation() {
+            dateText += " (\(timeZoneAbbreviation))"
+        }
+
+        return dateText
+    }
```

Comment:
I did some more research. We can simply use `timeStyle: .long` to include to time zone - no manual string concatenation needed :)

Reply:
Alright, thanks for the note, I'll adjust that:)